.github/workflows/helm_docs.yaml

```diff
@@ -0,0 +1,31 @@
+name: Helm docs
```

Comment:
We don't have this mechanism in other repositories and I am not sure about this. Can we remove it from this PR and start a separate discussion? It will introduce a maintenance burden and we need to ask others' opinions.